app/controllers/projects_controller.rb

```diff
@@ -40,17 +41,12 @@ def show
   end
 
   def destroy
-    if @project.reports.empty?
-      if @project.destroy
-        flash[:success] = t ".delete_success"
-        redirect_to projects_path
-      else
-        flash[:danger] = t ".fail_delete"
-        redirect_to root_path, status: :unprocessable_entity
-      end
+    if @project.destroy
+      flash[:success] = t ".delete_success"
+      redirect_to projects_path
     else
-      flash[:danger] = t ".delete_fail"
-      redirect_to @project, status: :unprocessable_entity
+      flash[:danger] = t ".fail_delete"
+      redirect_to root_path, status: :unprocessable_entity
```

Comment:
```suggestion
      redirect_to projects_path, status: :unprocessable_entity
```
e nghĩ xoá được hay ko thì cx ko nên redirect về tận `root_path`